docs/ramalama-perplexity.1.md

```diff
@@ -0,0 +1,44 @@
+% ramalama-perplexity 1
+
+## NAME
+ramalama\-perplexity - measure perplexity of specified AI Model
```

Comment:
I have no idea what perplexity is, can you describe it in this command?  When would a user want to run this command?

Reply:
Can you add information from: https://github.com/ggerganov/llama.cpp/blob/master/examples/perplexity/README.md

Reply:
Added more info